src/coreclr/jit/emitxarch.cpp

```diff
@@ -4391,7 +4416,7 @@ inline UNATIVE_OFFSET emitter::emitInsSizeSV(instrDesc* id, code_t code, int var
 static bool baseRegisterRequiresSibByte(regNumber base)
 {
 #ifdef TARGET_AMD64
-    return base == REG_ESP || base == REG_R12;
+    return base == REG_ESP || base == REG_R12 || base == REG_R20 /*|| base == REG_R28*/;
```

Comment:
any reason why we want to keep these comments?

Reply:
Removed.